sdk/communication/azure-communication-rooms/README.md

```diff
@@ -78,11 +78,14 @@ participant2 = new RoomParticipant(new CommunicationUserIdentifier("<ACS User MR
 participants.add(participant1);
 participants.add(participant2);
 
+// Starting in 1.1.0-beta.1 release, `PstnDialOutEnabled` property is added to enable or disable PSTN Dial-Out feature in a room. The `PstnDialOutEnabled` is an optional property. If `PstnDialOutEnabled` is not provided, then the default for `PstnDialOutEnabled` is false.
```

Comment:
We generally don't include the version in which we included the feature in samples. This information should be part of the CHANGELOG.md file.

Reply:
Updated